app/components/grades/gradeInfo.js

```diff
@@ -1,38 +1,31 @@
-const _ = require('lodash');
+import _ from 'lodash';
 
-const getPersonalInfo = student => ({
-    name:   `${student.lastname}, ${student.firstname}`,
-    gender: student.gender === 'male' ? 'M' : 'F'
-});
+function getPersonalInfo(student) {
+    return {
+        name:   `${student.lastname}, ${student.firstname}`,
+        gender: student.gender === 'male' ? 'M' : 'F'
+    };
+}
 
-const checkGradeId = grade => {
+function checkGradeId(grade) {
     if (_.isUndefined(grade)) {
         return null;
     }
 
     return grade._id;
```

Comment:
@hcuffy I don't know what JS version you are using , but in the newest you could do something like 

`return grade?._id`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
fixed